lib/operationSpec.ts

```diff
@@ -14,6 +16,18 @@ export interface OperationSpec {
    */
   httpMethod: HttpMethods;
 
+  /**
+   * The URL that was provided in the service's specification. This will still have all of the URL
+   * placeholder variables in it.
```

Comment:
I'd be tempted to use the word "template" over "placeholder"

Reply:
Sure. I'll change it.